crates/symbolicator/src/utils/paths.rs

```diff
@@ -357,7 +358,7 @@ fn get_unified_path(filetype: FileType, identifier: &ObjectId) -> Option<String>
         }
         FileType::Breakpad => "breakpad",
         FileType::SourceBundle => "sourcebundle",
-        FileType::PList => "plist",
+        FileType::BcSymbolMapUuidMap => "plist",
```

Comment:
why not change it here?

Reply:
This was an excellent question and made me re-think the entire naming.  😄 

The short answer for this was that this defines the filename used in the unified symbol server layout.  I've now renamed the entire thing to be consistently `uuidmap`, with the motivation written in the `FileType::UuidMap` doc comment.  Basically the tradeoff I'm making it to allow it to evolve for MachO UUID mappings, but not for other executable formats.  I'm also decoupling the name from BCSymbolMap as it can easily be more generic if they introduce something else.